### ISSUE TITLE
markdown.toc generate wrong toc while using [[toc]]

### ISSUE THREAD

**Issue by chuxubank** [vuejs/vuepress#1450]
- [x] I confirm that this is an issue rather than a question.

## Bug report

My content:
``` markdown
# MVC; iOS, Xcode and Swift Demonstration
[[toc]]
```

My config:
```js
markdown: {
        toc: {
            includeLevel: [3, 4, 5]
        }
}
```

#### Version
vuepress/1.0.0-alpha.44 darwin-x64 node-v11.11.0
#### Steps to reproduce

[This markdown file](https://github.com/chuxubank/Learning-iOS/blob/master/Developing-iOS-11-Apps-with-Swift/L2-MVC-iOS-Xcode-Swift-Demonstration/README.md)

[Whole Project](https://github.com/chuxubank/Learning-iOS)

#### What is expected?

Before `1.0.0-alpha.40`
<img width="708" alt="image" src="https://user-images.githubusercontent.com/17001575/54419354-312da780-4742-11e9-9a2f-c37f1c2e876a.png">

#### What is actually happening?

After `1.0.0-alpha.40`
<img width="729" alt="image" src="https://user-images.githubusercontent.com/17001575/54419447-6a661780-4742-11e9-8738-bf1369dd45b8.png">
Not deployed yet.

#### Other relevant information

- Your OS: macOS
- Node.js version: v11.11.0
- Browser version: Safari 12.0.3
- Is this a global or local install?  global
- Which package manager did you use for the install? yarn
- Does this issue occur when all plugins are disabled? yes

**Comment by Shigma** [comment]
Known issue. VuePress currently does not support TOC with level greater than 3.

**Comment by aaronlippold** [comment]
Is there a place we could help support resolving this issue or is it already in the work stream of tasks for 1.0 release.

**Comment by chuxubank** [comment]
fixed in `1.0.3`
